Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/contests/contest-solution-submit/ContestSolutionSubmitPage.tsx

```diff
@@ -649,11 +666,10 @@ const ContestSolutionSubmitPage = () => {
                     <span className={styles.title}>Submissions</span>
                     <IoMdRefresh
                       size={30}
-                      onClick={() => getSubmissionsData({
-                          id: Number(selectedContestDetailsProblem!.id),
-                          page: selectedSubmissionsPage,
-                          isOfficial: isCompete,
-                      })}
+                      className={isRotating
```

Comment:
A toggle reading "Refresh" will also be very nice

Reply:
Added a button reading "Refresh", which servers to better showcase that the button is used to refresh the submissions.